seed.js

```diff
@@ -0,0 +1,296 @@
+    'https://i.etsystatic.com/9257791/r/il/2e3c37/1952714655/il_794xN.1952714655_n9xz.jpg',
+    'https://i.etsystatic.com/12461585/r/il/20ff1e/1903805394/il_794xN.1903805394_imm3.jpg',
+    'https://i.etsystatic.com/11053221/r/il/75c4b1/1667418358/il_794xN.1667418358_o7ea.jpg',
+    'https://i.etsystatic.com/9846259/r/il/556129/1009303514/il_794xN.1009303514_rfxl.jpg',
+    'https://i.etsystatic.com/7878138/r/il/354a36/1870845106/il_794xN.1870845106_fyzw.jpg',
+    'https://i.etsystatic.com/17897135/r/il/ac5ceb/1869678418/il_fullxfull.1869678418_ndos.jpg',
+    'https://i.etsystatic.com/20624746/r/il/1bc884/2012087262/il_794xN.2012087262_8bqn.jpg',
+    'https://i.etsystatic.com/11451843/r/il/fec435/1802568295/il_794xN.1802568295_2ql3.jpg',
+    'https://i.etsystatic.com/20748570/r/il/b89624/1954579878/il_794xN.1954579878_98qu.jpg',
+    'https://i.etsystatic.com/17212209/r/il/7d8a4f/1502749754/il_794xN.1502749754_6qxv.jpg',
+    'https://i.etsystatic.com/14866640/r/il/686377/1484186786/il_794xN.1484186786_gzr2.jpg',
+    'https://i.etsystatic.com/13852279/r/il/861bef/1507989383/il_fullxfull.1507989383_ia74.jpg',
+    'https://i.etsystatic.com/11159473/r/il/8d6840/1332712953/il_fullxfull.1332712953_h6x2.jpg',
+    'https://i.etsystatic.com/11848782/r/il/4c3a86/2062699229/il_794xN.2062699229_t3ig.jpg',
+    'https://i.etsystatic.com/7991311/r/il/47c3dd/1762846385/il_794xN.1762846385_iqyu.jpg',
+    'https://i.etsystatic.com/10670358/r/il/2ca2d6/1610685447/il_fullxfull.1610685447_czb6.jpg',
+    'https://i.etsystatic.com/10071613/r/il/d8c943/1356974013/il_794xN.1356974013_m4sp.jpg'
+  ];
+
+
+  return randomPhoto[counter];
+};
+
+
+//gets a random date
+let getRandomDate = () => {
+  let months = [
+    'Jan',
+    'Feb',
+    'Apr',
+    'May',
+    'June',
+    'July',
+    'Aug',
+    'Sept',
+    'Oct',
+    'Nov',
+    'Dec'
+  ];
+
+
+  let randomMonth = months[Math.floor(Math.random() * months.length)];
+
+  let randomDay = Math.floor(Math.random() * 31);
+
+  let randomYear = Math.floor(Math.random() * 2) + 2017;
+
+  return `${randomMonth} ${randomDay}, ${randomYear}`;
+};
+
+
+//creates all the 25 comments into an array
+let getComments = () => {
+  let comments = [];
+
+
+  for (let i = 0; i < 25; i++) {
+    let currentComment = {
+      id: i,
+      reviewerName: getRandomNames(),
+      reviewerAvatar: getRandomAvatar(),
+      reviewerComment: getRandomComment(),
+      reviewerPhotoInComment: getRandomPhotoInComment(),
+      reviewerItem: getRandomReviewerItem(),
+      rating: getRandomRatings(),
+      reviewerDate: getRandomDate()
+    };
+    counter++;
+    comments.push(currentComment);
+
+  }
+
+
+  //return an array of 25
+  return comments;
+};
+
+
+
+//creating each object for the collection
+let seeds = [];
+for (let i = 0; i < 100; i++) {
+  let seedSeller = {
+    seller: getRandomNames(),
+    reviews: getRandomReviews(),
+    ratings: getRandomRatings(),
+    comments: getComments()
+  };
+
```

Comment:
Consider adding a sellerId number as a property, its always better to be able to do look ups by number if possible. If you are generating one automatically then forget I said this haha.

Reply:
Lol Yeah, Its generating one automatically.